packages/react-components/react-divider/src/stories/DividerCustomStyles.stories.tsx

```diff
@@ -21,7 +21,6 @@ const useStyles = makeStyles({
     display: 'flex',
     flexDirection: 'column',
     justifyContent: 'center',
-    backgroundColor: 'white',
```

Comment:
Removed it as it causes really bad problems for other themes.

### HC

![image](https://user-images.githubusercontent.com/14183168/170248257-c1b68b0e-8ca6-49aa-a885-f15ded508ebf.png)

### Dark 

![image](https://user-images.githubusercontent.com/14183168/170248291-453b9b4a-7078-4466-aa65-aaea7958488c.png)